lib/avromatic/model/serialization.rb

```diff
@@ -1,30 +1,24 @@
-require 'avro_turf/messaging'
 require 'avromatic/model/passthrough_serializer'
 
 module Avromatic
   module Model
 
-    # This concern adds support for serialization to a model
-    # generated from Avro schema(s).
+    # This module provides serialization support for encoding directly to Avro
+    # without dependency on a schema registry.
     module Serialization
```

Comment:
Should we rename this module to something like `Raw` or `RawSerialization`? If we go with the latter we should probably also rename `Messaging` to `MessagingSerialization` for consistency.